mLRS/Common/fhss.h

```diff
@@ -742,7 +749,10 @@ class tFhss
 #endif
     }
 
-    // only used by tx cli
+    //-- Tx module only
+
+    uint8_t Cnt(void) { return fhss1stBand.Cnt(); }
+    uint8_t CurrI_4mBridge(void) { return fhss1stBand.CurrI_4mBridge(); }
```

Comment:
Should mBridge provide a way to specify which band we want stats on?

Reply:
maybe at some point. Not now.